packages/utilities/fast-colors/package.json

```diff
@@ -17,10 +17,11 @@
   },
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
+  "type": "module",
   "scripts": {
-    "build": "tsc -p ./tsconfig.build.json && yarn doc",
+    "build": "tsc -p ./tsconfig.build.json",
```

Comment:
Just want to confirm we only want to run `yarn doc` in test scenarios. I'm fine with this but wanted to confirm.

Reply:
Oops no, good catch. I forget to put that back